reference/reorder/rcm_kernels.cpp

```diff
@@ -255,7 +255,7 @@ void get_permutation(std::shared_ptr<const ReferenceExecutor> exec,
             ++tail_offset;
         }
 
-        // Get the neigbours of the next vertex,
+        // Get the neighbours of the next vertex,
```

Comment:
There is an "h" missing.

Reply:
I changed to US spelling in the comments, however, both variants are used across the rcm_kernels files. Changing it would also be an interface break.